### DISCUSSION TITLE
Welcome to Acala Discussions!

### DISCUSSION THREAD

**Discussion by xlc** [AcalaNetwork/Acala#616] [0. General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by 0xthreebody** [comment]
Acala Github wiki: <https://github.com/AcalaNetwork/Acala/wiki>  
Acala Wiki: <https://wiki.acala.network/>  
API Manual: <https://developer.acala.network/>

**Comment by 0xthreebody** [comment]
## General Links

### Info

* [Website](https://acala.network/)
* [E-mail](mailto:hello@acala.network)
* [Twitter](https://twitter.com/AcalaNetwork)
* [Medium](https://medium.com/acalanetwork)
* [Telegram](https://t.me/AcalaOfficial)
* [Telegram Announcement](https://t.me/AcalaAnnouncement)
* [Discord](https://discord.gg/9YEgHs8)
* [Riot](https://riot.im/app/#/room/#acala:matrix.org)
* [Discourse Forum](https://acala.discourse.group/)
* [GitHub](https://github.com/AcalaNetwork)
* [Wiki](https://wiki.acala.network/)
* [Newsletter](https://share.hsforms.com/1X9RxkXk-R62I0VNbATaDXw4h8qc)
* [PolkaDeFi](https://polkadefi.acala.network/)

### Literature

* [&lt;Acala Whitepaper&gt;](https://github.com/AcalaNetwork/Acala-white-paper/blob/master/Acala_Whitepaper.pdf)
* [&lt;Acala Token Economy Paper&gt;](https://github.com/AcalaNetwork/Acala-white-paper/blob/master/Acala_Token_Economy_Paper.pdf)
* [&lt;Building a Decentralized Sovereign Wealth Fund&gt;](https://github.com/AcalaNetwork/Acala-white-paper/blob/master/Building_a_Decentralized_Sovereign_Wealth_Fund.pdf)

### UI and explorers

* [Dapp](https://apps.acala.network/)
* [Console Portal](https://console.acala.network/)
* [Telemetry](http://telemetry.polkadot.io/)
* [Polkawallet](https://polkawallet.io/)
* [Subscan](https://acala-testnet.subscan.io/)
* [Polkadot{.js} extension ****Chrome](https://chrome.google.com/webstore/detail/polkadot%7Bjs%7D-extension/mopnmbcafieddcagagdcbnhejhlodfdd?hl=en%20Firefox%20Store%20%EF%BC%9Ahttps://addons.mozilla.org/en-US/firefox/addon/polkadot-js-extension/)

###  Events Recording

* [Polkadot Decoded - Acala’s Cross-Chain DeFi Hub, Stablecoin & Decentralized Sovereign Wealth Fund - Ruitao Su](https://www.crowdcast.io/e/polkadot-decoded/6)
* [PolkaDeFi Day - The Polkadot Decentralized Finance Community Conference](https://www.crowdcast.io/e/polkadefi-conference?utm_campaign=discover&utm_source=crowdcast&utm_medium=discover_web)
* [Web3 Builders: Laminar & Acala](https://www.crowdcast.io/e/polkadot-web3-builders-laminar-acala?utm_campaign=discover&utm_source=crowdcast&utm_medium=discover_web)
* [Web 3.0 Bootcamp Opening Ceremony - Live Stream](https://www.crowdcast.io/e/f2ykih2r?utm_campaign=discover&utm_source=crowdcast&utm_medium=discover_web)
* [DeFi discussion](https://www.crowdcast.io/e/defi-discussions/52)
* [Acala & Laminar \| Crypto Interview \| BlockchainBrad](https://www.youtube.com/watch?v=kCVJojzYomo&t=2014s)
* [Introducing Acala - Cross-chain Open Finance Platform](https://www.youtube.com/watch?v=jpF7ncfnmxM)
* [Web 3.0 Bootcamp Online Opening Ceremony](https://www.bilibili.com/video/BV1ng4y167QB)
* [Ready Layer One](https://readylayer.one/)
* [SF BLockchain Week - Unitize](https://unitize.online/speakers)
* [Acala Joining Product Validation Day with IDEO CoLab Ventures](https://medium.com/acalanetwork/acala-joining-product-validation-day-with-ideo-colab-ventures-3735f9592cf6)
* [Figment Network’s Staking Hub hosted an AMA for Acala on July 2, where we answered questions on Acala’s DeFi hub position in Polkadot](https://twitter.com/StakingHub/status/1276194861838974976?s=20)
* [Hackusama — the very first Polkadot / Kusama hackathon is now LIVE with a deadline on Aug 14](https://polkadot.network/join-hackusama-hack-on-polkadots-wild-cousin/)
